src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,57 +70,176 @@ contract DssSpellCollateralAction {
-
-        // DssExecLib.addNewCollateral(
-        //     CollateralOpts({
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    uint256 constant ZERO_PCT_RATE                  = 1000000000000000000000000000;
+
+    // --- Math ---
+    uint256 public constant WAD = 10**18;
+
+    // -- RWA007 MIP21 components --
+    address constant RWA007                         = 0x02091C59C92fc5E0575b3B133c3caa4f57D79b0e;
+    address constant MCD_JOIN_RWA007_A              = 0xe3BA7B6c922cb4622AfF00C2e2A467fF6A06CA29;
+    address constant RWA007_A_URN                   = 0x78D9ccbe652E2aE7B11289B4F202278058aEfeeA;
+    address constant RWA007_A_JAR                   = 0xcF8A6445B6e22A8e2Fe9d7bd21410bB7a8b8D054;
+    // Goerli: Coinbase / Mainnet: Coinbase
+    address constant RWA007_A_OUTPUT_CONDUIT        = 0x9B4Af5F55b23cC2D86f58d7c08cD025bA1901013;
+    // Jar and URN Input Conduits
+    address constant RWA007_A_INPUT_CONDUIT_URN     = 0xFC80d4037dDadDb380e752AE46Aca262eEb812fC;
+    address constant RWA007_A_INPUT_CONDUIT_JAR     = 0x9ca5F7b1F5DCC287657c20547176a6733EB2c046;
+
+    // MIP21_LIQUIDATION_ORACLE params
+    string  constant RWA007_DOC                     = "QmRe77P2JsvQWygVr9ZAMs4SHnjUQXz6uawdSboAaj2ryF"; // TODO
+    // There is no DssExecLib helper, so WAD precision is used.
+    uint256 constant RWA007_A_INITIAL_PRICE         = 250_000_000 * WAD;
```

Comment:
this should be set following current DC set of 1M (checking previous patterns)

Reply:
We plan to adjust DC to 250mm after first draw, so i am not sure we should change this

Reply:
if this will be done in a following spell, the DC increase would also require a `price` bump (see [here](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L661))
cc: @godsflaw

Reply:
After we make this test with 1MM, we are going to enable `autoLine` with 50MM steps up to 250MM.

I don't see the reason why the price should mirror the debt ceiling, would you mind to please explain?

Reply:
I think future spell actions on `RWA007` are out of scope for the current spell.
In case of enabling Autoline in future spells for `RWA007`, the `price` could be bumped to Autoline Max DC along with autoline activation.

Reply:
> I don't see the reason why the price should mirror the debt ceiling, would you mind to please explain?

Reply:
I don't have a strong opinion, as @godsflaw think it's the best approach here.

Reply:
Is the collateral price actionable? Like could someone take a malicious action of printing more DAI before our next spell? If so that's a blocker on my end, otherwise I think it's somewhat sensible to put the price in higher so we don't have to redo work at a future date.

Reply:
It's not actionable in this case because the debt ceiling would prevail.
It only affects the collateralization ration, but since we don't have automatic liquidations for RWAs, it shouldn't matter.

Reply:
Well will leave any unthought of technical reason to PE, but seems like it should be okay from my end

Reply:
I would set the price to what we expect the facility to eventually accommodate, as resetting the price is non-standard and should only be reserved for when we absolutely need to (we've made mistakes in the past).  The trickle should happen with the autoline or debt ceiling increases if possible.

This path also more accurately captures the "value" of the RWA-007 token and the amount of debt taken against it.